### PR TITLE
VREffect: actually exit VR for deprecated API (originally from #8570)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: examples/js/effects/VREffect.js [hunk 1]
```diff
@@ -148,22 +149,25 @@ THREE.VREffect = function ( renderer, onError ) {
 
 		requestFullscreen = 'requestFullscreen';
 		fullscreenElement = 'fullscreenElement';
+		exitFullscreen = 'exitFullscreen';
```
Comment (cvan) [inline examples/js/effects/VREffect.js]: paging @brianchirls

**Comment by mrdoob** [comment]
Looks good @brianchirls?

**Comment by brianchirls** [comment]
Thanks. Will take a look this afternoon.

**Comment by mrdoob** [comment]
Looks good to me actually.

**Comment by mrdoob** [comment]
Thanks!

**Comment by brianchirls** [comment]
Hey gang. I'm sorry I wasn't able to get on this earlier today. Been dealing with some family stuff.

Adding `exitFullscreen` is great. And thanks for cleaning up my style screw-ups.

I have one problem with this PR. As discussed over in #8568, Firefox Nightly on Windows currently has a bit of a bug with firing the `fullscreenchange` event. For that reason, I very intentionally had this code listen on both the prefixed and un-prefixed event. This PR undoes that. @cvan said he was going to file a bugzilla ticket. Until that gets fixed, I think we should revert that part of the code.

When the bug does eventually get fixed, the `addEventListener` code should not be inside every `if` block. It should match the pattern for the rest of the properties/methods where you set the string value and use it once when you need it.

**Pull request merged**